overlord/devicestate/devicemgr.go

```diff
@@ -547,7 +547,7 @@ func (m *DeviceManager) ensureInstalled() error {
 
 	var seeded bool
 	err := m.state.Get("seeded", &seeded)
-	if err != nil {
+	if err != nil && err != state.ErrNoState {
```

Comment:
nice! and funny, I changed this in #8690 too to avoid some spam in the logs during install